### ISSUE TITLE
Color Contrast: Follow Button

### ISSUE THREAD

**Issue by AutoSponge** [Hashnode/user-feedback#172]
**Describe the bug**
Element has insufficient color contrast of 4.29 (foreground color: `#2962ff`, background color: `#e3f2fd`, font size: 10.5pt (14px), font weight: bold). 

**To Reproduce**
Element location: `.hn-user-card.flex-column.flex-shrink-0:nth-child(1) > .hn-follow-button > span`
Element source: `<span>Follow</span>`

**Expected behavior**
Expected contrast ratio of 4.5:1

Fix Options:
- Change text-color to `#295CFF` or darker.
- Change background-color to `#CCFFFF` or lighter.

**Comment by BolajiAyodeji** [comment]
Hi @AutoSponge,

Thanks for the amazing feedback. We are working on getting this fixed and we'll update you soon.
